packages/client/components/DashboardAvatars/DashboardAvatars.tsx

```diff
@@ -1,94 +1,142 @@
 import DashboardAvatar from './DashboardAvatar'
 
-const desktopBreakpoint = makeMinWidthMediaQuery(Breakpoint.SIDEBAR_LEFT)
-
-const AvatarsList = styled('div')({
+const AvatarsList = styled('div')<{isDesktop: boolean}>(({isDesktop}) => ({
   display: 'flex',
-  justifyContent: 'flex-start',
-  marginTop: 16,
-  width: '75%',
-  [desktopBreakpoint]: {
-    justifyContent: 'flex-end',
-    marginTop: 0,
-    width: '100%'
-  }
-})
+  flexDirection: 'column',
+  marginRight: 6,
+  position: 'relative',
+  // AvatarsWrapper left causes avatars to move into left padding on mobile by 4px (-2px for the transparent border)
+  left: isDesktop ? 0 : 2
+}))
 
-const ScrollContainer = styled('div')({
+const AvatarsWrapper = styled('div')({
   display: 'flex',
-  maxWidth: '100%',
-  overflow: 'hidden'
-})
-
-const ItemBlock = styled('div')({
-  marginRight: 8,
+  justifyContent: 'center',
   position: 'relative',
-  [desktopBreakpoint]: {
-    marginBottom: 8,
-    marginLeft: 8,
-    marginRight: 0
-  }
+  left: -4 // each avatar is given 20px of width but the final avatar uses 28px
 })
 
-interface Props {
-  team: DashboardAvatars_team
-}
+const OverflowWrapper = styled('div')({
+  width: ElementWidth.DASHBOARD_AVATAR_OVERLAPPED
+})
 
 const OverflowCount = styled('div')({
   alignItems: 'center',
+  border: `2px solid ${PALETTE.SLATE_200}`,
   backgroundColor: PALETTE.SKY_400,
   borderRadius: '50%',
   display: 'flex',
-  height: 32,
+  height: ElementHeight.DASHBOARD_AVATAR,
   justifyContent: 'center',
   color: '#fff',
   fontSize: 12,
   fontWeight: 600,
   overflow: 'hidden',
   userSelect: 'none',
-  width: 32
+  width: ElementWidth.DASHBOARD_AVATAR,
+  '&:hover': {
+    cursor: 'pointer'
+  }
+})
+
+const StyledButton = styled(PlainButton)({
+  fontSize: 12,
+  height: 16,
+  lineHeight: '16px',
+  fontWeight: 600,
+  color: PALETTE.SLATE_700,
+  textAlign: 'center',
+  width: '100%',
+  WebkitTapHighlightColor: 'transparent',
+  '&:hover': {
+    cursor: 'pointer'
+  }
 })
 
+interface Props {
+  team: DashboardAvatars_team
+}
+
+type Avatar = DashboardAvatars_team['teamMembers'][0]
+
 const DashboardAvatars = (props: Props) => {
   const {team} = props
-  const {id: teamId, isLead: isViewerLead, teamMembers} = team
-  const wrapperRef = useRef<HTMLDivElement>(null)
-  const avatarsRef = useRef<HTMLDivElement>(null)
-  const maxAvatars = useAvatarsOverflow(wrapperRef, avatarsRef)
+  const {id: teamId, teamMembers} = team
+  const isDesktop = useBreakpoint(Breakpoint.SIDEBAR_LEFT)
+  const atmosphere = useAtmosphere()
+  const {viewerId} = atmosphere
+  const maxAvatars = isDesktop ? 10 : 6
   const overflowCount = teamMembers.length > maxAvatars ? teamMembers.length - maxAvatars + 1 : 0
-  const visibleAvatars = overflowCount === 0 ? teamMembers : teamMembers.slice(0, maxAvatars - 1)
+  const sortedAvatars = useMemo(() => {
+    const visibleAvatars = overflowCount === 0 ? teamMembers : teamMembers.slice(0, maxAvatars - 1)
```

Comment:
you'll want to slice after you do the sort.
since slice is mutative, i'd do something like 
```ts
teamMembers.slice().sort((a,b) => doStuff(a,b)).slice(0, maxAvatars -1)
```

Reply:
I think that splice is mutative, but [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) is non-mutative - it returns a new array? I think your suggestion of slicing after the sort definitely makes sense though, as we want to sort the entire array, not just the first 10 avatars. I've pushed an update with: `teamMembers.sort((a,b) => doStuff(a,b)).slice(0, maxAvatars -1)`